WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -11,6 +11,12 @@ final class SiteAddressViewController: LoginViewController {
     @IBOutlet private weak var tableView: UITableView!
     @IBOutlet var bottomContentConstraint: NSLayoutConstraint?
 
+    override var sourceTag: WordPressSupportSourceTag {
+        get {
+            .loginSiteAddress
+        }
```

Comment:
nit: could it be simplified:

```suggestion
        .loginSiteAddress
```

it's also interesting that the `loginSiteAddress` case exists before but doesn't seem to be used before this PR?

Reply:
Simplified in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/672/commits/9ea47eca983e49daf6363e0bd6cbbe0286f16a91

> it's also interesting that the loginSiteAddress case exists before but doesn't seem to be used before this PR?

It is used [here](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/trunk/WordPressAuthenticator/Signin/LoginSiteAddressViewController.swift#L11-L15), and it seems like the screen isn't being used anymore. I believe that this screen and other screens from that flow were deprecated after the unified login project. Internal ref - p91TBi-2FR-p2